src/main/java/Duke.java

```diff
@@ -1,10 +1,101 @@
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
+    public static final int MAX_TASKS = 100;
+
+    public static String getFirstWord(String word){
+        String firstWord = word;
+        if (firstWord.contains(" ")) {
+            firstWord = firstWord.substring(0, firstWord.indexOf(" "));
+        }
+        return firstWord;
+    }
+
+    public static boolean checkOneWord(String word){
+        String [] wordArray = word.trim().split(" ");
+        if(wordArray.length == 1){
+            return true;
+        } else{
+            return false;
+        }
+    }
+    public static void main(String[] args) throws DukeException {
```

Comment:
You should apply SLAP here and abstract the implementation details into different methods in order to make your code more readable.